actionview/lib/action_view/helpers/number_helper.rb

```diff
@@ -100,6 +100,9 @@ def number_to_phone(number, options = {})
       #   absolute value of the number.
       # * <tt>:raise</tt> - If true, raises +InvalidNumberError+ when
       #   the argument is invalid.
+      # * <tt>:strip_insignificant_zeros</tt> - If +true+ removes
+      #   insignificant zeros after the decimal separator (defaults to
+      #   +false+).
```

Comment:
Can you explain a little more what "insignificant" means in this documention?

Reply:
I think "(...)insignificant zeros after the decimal separator(...)" is kind of clear enough, but I can add something like, "(...)insignificant zeros after the decimal separator, that do not affect the value of a number and may be omitted(...)". What do you think @eileencodes ?

Reply:
@eileencodes what do you think? 👆